roles/tests/test_api.py

```diff
@@ -20,7 +23,8 @@ class TestRoleApi(LoreTestCase):
     roles_init_new_repo is tested indirectly from
     the repository model where it is called
     """
-    # pylint: disable=too-many-instance-attributes
+    # there are too many self assignment in the init: disabling the check
+    # pylint: disable=too-many-instance-attributes, invalid-name
```

Comment:
If the too many instance attributes only happens in init, you should scope the check to that function, as disables should be as close to the problem as you can be.  The invalid name, can stay at the class scope though.